package.json

```diff
@@ -18,7 +18,9 @@
     "dev-firefox": "make dev browser=firefox type=dev",
     "release-firefox": "make browser=firefox type=release",
     "dev-chrome": "make dev browser=chrome type=dev",
-    "release-chrome": "make browser=chrome type=release && make chrome-release-zip"
+    "beta-firefox": "npm run release-firefox && cd build/firefox/release && npm run remove-firefox-id && web-ext build",
```

Comment:
so if I'm reading this right.. this is way too convoluted.

- invoke node.js in order to run `make`
- then change the directory
- then invoke node.js again in order to run a shell command 
- which runs `sed` in order to update something in a json file
- then run `web-ext build`

shouldn't this be in one shell script, or one single makefile?